changelog_unreleased/typescript/12390.md

```diff
@@ -1,7 +1,7 @@
 #### Print trailing comma for rest elements in tuple types (#12390 by @sosukesuzuki)
 
-TypeScript allows to put rest elements in places other than the end of tuple types since [4.2](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-2.html#leadingmiddle-rest-elements-in-tuple-types).
-Prettier now prints a trailing comma for the trailing rest element as the it may be followed by the next element.
+TypeScript has allows rest elements in tuple types (`...T`) to have normal elements come after them since [TypeScript 4.2](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-2.html#leadingmiddle-rest-elements-in-tuple-types).
```

Comment:
I suggested the wrong tense here, sorry!

```suggestion
TypeScript has allowed rest elements in tuple types (`...T`) to have normal elements come after them since [TypeScript 4.2](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-2.html#leadingmiddle-rest-elements-in-tuple-types).
```